### ISSUE TITLE
setCurrentScreen is deprecated api:firebase-analytics

### ISSUE THREAD

**Issue by Prabha10cs** [firebase/quickstart-android#1178]
'setCurrentScreen(android.app.Activity, java.lang.String, java.lang.String)' is deprecated

**Comment by vincent-paing** [comment]
Yes you are supposed to use `FirebaseAnalytics.logEvent` 

Per [this blog post](https://firebase.googleblog.com/2020/08/google-analytics-manual-screen-view.html)

```
@Override
public void onResume() {
    super.onResume();

    // After enough time has passed to make this screen view significant.
    Bundle bundle = new Bundle();
    bundle.putString(FirebaseAnalytics.Param.SCREEN_NAME, screenName);
    bundle.putString(FirebaseAnalytics.Param.SCREEN_CLASS, screenClass);
    bundle.putString(MyAppAnalyticsConstants.Param.TOPIC, topic);
    mFirebaseAnalytics.logEvent(FirebaseAnalytics.Event.SCREEN_VIEW, bundle);
}
```

**Comment by samtstern** [comment]
Thanks for noticing this!  Preparing a PR to fix this now.

**Comment by samtstern** [comment]
Fixed by @rosariopfernandes  in #1176
